### PR TITLE
Map Updates: Arena

### LINKED ISSUE #2122: Station Anchor room in Arena missing a tile, starts and stays spaced
## Description

The station anchor room in Arena starts spaced. There's a missing tile under the center of the station anchor. Additionally, the room stays PERMANENTLY spaced even after to fix it, leading to the entire engi department showing up red on the atmos console. 
**Reproduction**

1. Load up the map Arena
2. Start round
3. Go to Station Anchor room
4. It stays space the entire round
**Screenshots**

N/A
**Additional context**

I learned this on round #16658 where a lot of other funky atmos shit was happening.

### LINKED ISSUE #2281: Arena shuttle is always in the wrong spot
## Description
arena shuttle always docks south instead of east, which causes evac to take forever.

**Reproduction**
go to arena and call evac

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by IamVelcroboy**
## About the PR
**Arena**
- Adds AI
- Remodels the medbay
- Moves/adds cameras around for better AI
- FIxes [#2122](https://github.com/DeltaV-Station/Delta-v/issues/2122)
- Adds robo airlocks and spawnpoint for [#2028](https://github.com/DeltaV-Station/Delta-v/pull/2028)
- Adds Morgue airlocks
- Places proper bio
- Fixes [#2281](https://github.com/DeltaV-Station/Delta-v/issues/2281)

## Why / Balance
🥊

## Technical details
n/a

## Media
image

## Requirements
n/a

## Breaking changes
n/a

**Changelog**
:cl: Velcroboy
- tweak: Changed Arena medbay layout

### CODE REVIEW

**Comment by MilonPL** [comment]
HUGE

**Comment by IamVelcroboy** [comment]
ooo buttholes, I forgot alarms

**Comment by deltanedas** [comment]
@IamVelcroboy forgot job slots for ai and roboticist

**Comment by IamVelcroboy** [comment]
> @IamVelcroboy forgot job slots for ai and roboticist

I thought that would be in a separate PR when those roles are merged? No?

**Comment by deltanedas** [comment]
if the roles are enabled before any maps have slots itll be impossible to play them when you ready up for them
if maps have slots before roles are enabled all that happens is it makes tests fail, theres no ingame consequence

**Comment by IamVelcroboy** [comment]
Makes sense

File: Resources/Prototypes/Maps/arena.yml [hunk 1]
```diff
@@ -61,12 +61,14 @@
             Reporter: [ 2, 2 ]
             ServiceWorker: [ 1, 3 ]
           #science
+            AI: [ 1, 1 ]
```
Comment (JustTNE) [inline Resources/Prototypes/Maps/arena.yml]: The job slot here is incorrect:

```suggestion
            StationAi: [ 1, 1 ]
```

**Review by JustTNE — CHANGES_REQUESTED** [review]

**Review by therealDLondon — APPROVED** [review]

**Pull request merged**